benchio.1 [stderr]   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 46, in <module>
benchio.1 [stderr]     main()
benchio.1 [stderr]   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 39, in main
benchio.1 [stderr]     raise RuntimeError()
benchio.1 [stderr] RuntimeError
benchio.1 [end (1)] $TMP/venv/benchio/bin/voir $SRC/milabench/tests/yoshua-benchio/main.py --bad [at 1990-01-01 00:00:00.000000]
benchio.0
=========
  * no training rate retrieved
  * Error codes = 1, 1
  * 1 exceptions found
    * 1 x RuntimeError
        | Traceback (most recent call last):
        |   File "$TMP/venv/benchio/bin/voir", line 8, in <module>
        |     sys.exit(main())
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/cli.py", line 128, in main
        |     ov(sys.argv[1:] if argv is None else argv)
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/phase.py", line 331, in __call__
        |     self._run(*args, **kwargs)
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/overseer.py", line 242, in _run
        |     set_value(func())
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/scriptutils.py", line 37, in <lambda>
        |     return lambda: exec(mainsection, glb, glb)
        |   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 46, in <module>
        |     main()
        |   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 39, in main
        |     raise RuntimeError()
        | RuntimeError
benchio.1
=========
  * no training rate retrieved
  * Error codes = 1, 1
  * 1 exceptions found
    * 1 x RuntimeError
        | Traceback (most recent call last):
        |   File "$TMP/venv/benchio/bin/voir", line 8, in <module>
        |     sys.exit(main())
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/cli.py", line 128, in main
        |     ov(sys.argv[1:] if argv is None else argv)
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/phase.py", line 331, in __call__
        |     self._run(*args, **kwargs)
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/overseer.py", line 242, in _run
        |     set_value(func())
        |   File "$TMP/venv/benchio/lib/python3.10/site-packages/voir/scriptutils.py", line 37, in <lambda>
        |     return lambda: exec(mainsection, glb, glb)
        |   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 46, in <module>
        |     main()
        |   File "$SRC/milabench/tests/yoshua-benchio/main.py", line 39, in main
        |     raise RuntimeError()
        | RuntimeError
